src/components/common/ShareStructure/ShareStructures.vue

```diff
@@ -69,8 +69,8 @@ export default class ShareStructures extends Mixins(CommonMixin) {
   @Action(useStore) setValidComponent!: (x: ActionKvIF) => void
 
   // Local propertiues
-  protected isEditing = false
-  protected showResolutionDateDialog = false
+  isEditing = false
+  showResolutionDateDialog = false
```

Comment:
I didn't fix all of these, but at one point in this ticket, I touched this file, so I fixed it.